packages/core/src/transaction/Clause.ts

```
@@ -228,25 +227,24 @@ class Clause implements TransactionClause {
      */
-    public static transferToken(
-        tokenAddress: Address,
+    public static transferVTHOToken(
         recipientAddress: Address,
-        amount: VTHO,
-        clauseOptions?: ClauseOptions
+        amount: VTHO
     ): Clause {
         if (amount.value.isFinite() && amount.value.isPositive()) {
+            const vthoAddress = Address.of(VTHO_ADDRESS);
             return this.callFunction(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)